### DISCUSSION TITLE
webgl支持

### DISCUSSION THREAD

**Discussion by ghostlhq** [antvis/G6#3721] [Q&A] [answered]
刚才有看到G现在支持WebGL渲染，但是G6的graph好像是自带canvas当前G的例子尝试通过getCanvas方法获取canvas但是貌似没有例子里的setRendere方法[按需引入渲染器->运行时切换](https://g-next.antv.vision/zh/docs/guide/diving-deeper/switch-renderer#%E8%BF%90%E8%A1%8C%E6%97%B6%E5%88%87%E6%8D%A2)
我看到其他issue说今年打算支持WebGL，请问一下目前的计划还打算支持吗？

**Comment by Yanyan-Wang** [comment, marked as answer]
目前 G6 发布的版本仍然使用的是 G 4.0，正在接入 G 5.0（支持 webgl），预计下半年 beta，2024.06 正式版本。
